index.bs

```diff
@@ -1143,6 +1146,25 @@ The following constraints define when a [=/transaction=] can be
 
 </div>
 
+To <dfn export>cleanup Indexed Database transactions</dfn> for the
+[=event loop=], run the following steps for each [=/transaction=]
+with a matching [=transaction/cleanup event loop=]:
```

Comment:
I'm still not thrilled with this. Pondering something like: "To __cleanup Indexed Database transactions__, run the following steps for each __transaction__ with __cleanup event loop__ matching the __event loop__:"

That's still awkward phrasing, though. 

(The spec needs to speak about transactions that span multiple threads, although each transaction is of course associated with just one. In most cases the spec is written either in the context of one thread, or in the context of the "database thread" and so dodges the complexity. It's never explicit about this, though.)